keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -1129,33 +1151,31 @@ such a scenario, the scheduler has to make those decisions based on
 outdated information, in particular when making one scheduling
 decisions affects the next decision.
 
-We need to investigate:
-- Whether this really is a problem in practice, i.e. identify
-  workloads and drivers where this problem occurs.
-- Whether introducing some simple modeling of capacity helps.
-- Whether prioritization of nodes helps.
-
-For a discussion around modeling storage capacity, see the proposal to
-add ["total capacity" to
-CSI](https://github.com/container-storage-interface/spec/issues/301).
-
-### "Total available capacity" vs. "maximum volume size"
-
-The CSI spec around `GetCapacityResponse.capacity` [is
-vague](https://github.com/container-storage-interface/spec/issues/432)
-because it ignores fragmentation issues. The current Kubernetes API
-proposal follows the design principle that Kubernetes should deviate
-from the CSI spec as little as possible. It therefore directly copies
-that value and thus has the same issue.
-
-The proposed usage (comparison of volume size against available
-capacity) works either way, but having separate fields for "total
-available capacity" and "maximum volume size" would be more precise
-and enable additional features like even volume spreading by
-prioritizing nodes based on "total available capacity"
-
-The goal is to clarify that first in the CSI spec and then revise the
-Kubernetes API.
+[Scale testing](https://github.com/kubernetes-csi/csi-driver-host-path/blob/d6d9639077691986d676984827ea4dd7ee0c5cce/docs/storage-capacity-tracking.md)
```

Comment:
For the 100 nodes test, it took 1775s to schedule the pods. Did you run the baseline with 100 nodes too so we can compare?

Reply:
No. I can do that, but I am not sure what kind of conclusion we want to draw from that.

How quickly volumes get provisioned will always depend on the specific storage solution and volume sizes. There is no guarantee that it will have "less than x percent overhead" compared to pods with no volumes.

Reply:
I just want to have an idea of how good/bad scheduling retries affect.

If it's too bad, we can use those number to push on the CSI community to provide "something" to improve the situation.

Reply:
I ran the test. The relative overhead for adding one volume was even lower than it was for 10 nodes (factor of 1.58 instead of 1.68), but I didn't run the tests often enough to be sure that this wasn't because of variation.

See https://github.com/kubernetes-csi/csi-driver-host-path/compare/d117785876889cd365bcf0837f68a65b9e91a4f2..f053a7b0c4b719a5808fc47fdb3eba9cdade2067 for that update of https://github.com/kubernetes-csi/csi-driver-host-path/pull/350.